SampleApp/SampleApp/Screens/CreatablesView.swift

```diff
@@ -66,15 +66,36 @@ struct GenericCreatableViews {
     let creatables: AnyView
 }
 
+struct RealifeTechViewConverter {
+
+    private let factories: [RLTContentType: RLTCreatableFactory]
+
+    init(factories: [RLTContentType: RLTCreatableFactory]) {
+        self.factories = factories
+    }
+
+    func convert(items: [RLTContentItem]) -> [GenericCreatableViews] {
+        var views: [GenericCreatableViews] = []
+        let creatables = RLTContentConverter().convert(factories: factories, items: items)
+        creatables.forEach {
```

Comment:
reuse existing RLTViewCreatable.unwrappedGenericView functionality to avoid risk of integrator breaking open/closed principle.

Reply:
Done! Thank you